### PR TITLE
adding reference about KMIP on quick-start guide

### PR DESCRIPTION
**Pull request by gbarideau**
On KMIP guide : 
* Modification on the title
* Clarify that KMIP works on non OVHcloud product

On quick-start guide : 
* Adding a reference to "Using KMS" and "KMIP" guide

### CODE REVIEW

File: pages/manage_and_operate/kms/kms-kmip/guide.en-gb.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Usage of KMIP protocol"
+title: "Connect a compatible product using KMIP protocol"
```
Comment (Y0Coss) [inline pages/manage_and_operate/kms/kms-kmip/guide.en-gb.md]:
```suggestion
title: "How to connect a compatible product using KMIP protocol"
```

File: pages/manage_and_operate/kms/kms-kmip/guide.fr-fr.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Utilisation du protocole KMIP"
+title: "Connecter un produit compatible en utilisant le protocole KMIP"
```
Comment (Y0Coss) [inline pages/manage_and_operate/kms/kms-kmip/guide.fr-fr.md]:
```suggestion
title: "Comment connecter un produit compatible en utilisant le protocole KMIP"
```

File: pages/manage_and_operate/kms/kms-kmip/guide.fr-fr.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ L'objectif de ce guide est de présenter l'utilisation du protocole KMIP ainsi q
 ### Description
 
 Le protocole [KMIP](https://www.oasis-open.org/committees/tc_home.php?wg_abbrev=kmip) (Key Management Interoperability Protocol) est un protocole conçu pour standardiser les communications avec un KMS.
-Ainsi tout produit supportant le KMIP (comme VMware vSphere, Veeam, Nutanix, etc.) peut nativement s'interfacer avec un KMS compatible KMIP comme le KMS OVHcloud.
+Ainsi tout produit supportant le KMIP (comme VMware vSphere, Veeam, Nutanix, etc.) peut nativement s'interfacer avec un KMS compatible KMIP comme le KMS OVHcloud, que ces produits soient hébergés par OVHcloud ou non
```
Comment (Y0Coss) [inline pages/manage_and_operate/kms/kms-kmip/guide.fr-fr.md]:
```suggestion
Ainsi tout produit supportant le KMIP (comme VMware vSphere, Veeam, Nutanix, etc.) peut nativement s'interfacer avec un KMS compatible KMIP comme le KMS OVHcloud, que ces produits soient hébergés par OVHcloud ou non.
```

File: pages/manage_and_operate/kms/quick-start/guide.en-gb.md [hunk 4]
```diff
@@ -380,6 +380,13 @@ The API returns the certificate in PEM:
 
 Copy the value of the **certificatePEM** field to a **client.cert** file.
 
+### Use the OVHcloud KMS
+
+Once your OVHcloud KMS set-up, there is two different way to use it :
```
Comment (Y0Coss) [inline pages/manage_and_operate/kms/quick-start/guide.en-gb.md]:
```suggestion
Once your OVHcloud KMS is set up, there are two different ways to use it:
```

File: pages/manage_and_operate/kms/quick-start/guide.en-gb.md [hunk 5]
```diff
@@ -380,6 +380,13 @@ The API returns the certificate in PEM:
...
+
+- Using the [Rest API](/pages/manage_and_operate/kms/kms-usage), if you want to manually use the API to encrypt or sign your data
```
Comment (Y0Coss) [inline pages/manage_and_operate/kms/quick-start/guide.en-gb.md]:
```suggestion
- Using the [Rest API](/pages/manage_and_operate/kms/kms-usage) if you want to manually use the API to encrypt or sign your data.
```

File: pages/manage_and_operate/kms/quick-start/guide.en-gb.md [hunk 6]
```diff
@@ -380,6 +380,13 @@ The API returns the certificate in PEM:
...
+- Using the [KMIP protocol](/pages/manage_and_operate/kms/kms-kmip), if you want to connect any KMIP compatible products with your OVHcloud KMS
```
Comment (Y0Coss) [inline pages/manage_and_operate/kms/quick-start/guide.en-gb.md]:
```suggestion
- Using the [KMIP protocol](/pages/manage_and_operate/kms/kms-kmip) if you want to connect any KMIP compatible product with your OVHcloud KMS.
```

File: pages/manage_and_operate/kms/quick-start/guide.fr-fr.md [hunk 7]
```diff
@@ -381,6 +381,14 @@ L'API renvoie le certificat au format PEM :
 
 Copiez la valeur du champ **certificatePEM** dans un fichier **client.cert**.
 
+### Utiliser le KMS OVHcloud
+
+Une fois votre KMS OVHcloud initialisé, il est possible de l'utiliser de deux manières différentes :
+
+- En utilisant les [API Rest](/pages/manage_and_operate/kms/kms-usage), si vous souhaitez utiliser manuellement les API pour chiffrer ou signer vos données
```
Comment (Y0Coss) [inline pages/manage_and_operate/kms/quick-start/guide.fr-fr.md]:
```suggestion
- En utilisant les [API Rest](/pages/manage_and_operate/kms/kms-usage), si vous souhaitez utiliser manuellement les API pour chiffrer ou signer vos données.
```

File: pages/manage_and_operate/kms/quick-start/guide.fr-fr.md [hunk 8]
```diff
@@ -381,6 +381,14 @@ L'API renvoie le certificat au format PEM :
...
+- En utilisant le [protocole KMIP](/pages/manage_and_operate/kms/kms-kmip), si vous souhaitez connecter n'importe quel produit compatible KMIP avec le KMS OVHcloud
```
Comment (Y0Coss) [inline pages/manage_and_operate/kms/quick-start/guide.fr-fr.md]:
```suggestion
- En utilisant le [protocole KMIP](/pages/manage_and_operate/kms/kms-kmip), si vous souhaitez connecter n'importe quel produit compatible KMIP avec le KMS OVHcloud.
```

File: pages/index.md [hunk 9]
```diff
@@ -2029,7 +2029,7 @@
         + [Getting started with OVHcloud Key Management Service (KMS)](manage_and_operate/kms/quick-start)
         + [Using the OVHcloud Key Management Service (KMS) with your data](manage_and_operate/kms/kms-usage)
         + [OVHcloud KMS Architecture overview](manage_and_operate/kms/architecture-overview)
-        + [Using KMIP to communicate with OVHcloud KMS](manage_and_operate/kms/kms-kmip)
+        + [Connect a compatible product using KMIP protocol with OVHcloud KMS](manage_and_operate/kms/kms-kmip)
```
Comment (Y0Coss) [inline pages/index.md]:
```suggestion
        + [How to connect a compatible product using KMIP protocol with OVHcloud KMS](manage_and_operate/kms/kms-kmip)
```

**Review by Y0Coss — COMMENTED** [review]
Can you please duplicate these suggestions to all files?

**Comment by gbarideau** [comment]
Done

**Comment by Y0Coss** [comment]
Thank you for your contribution @gbarideau

**Pull request merged**